src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -51,8 +63,8 @@ class ActivityFeed extends React.Component<Props, State> {
     }
 
     componentDidUpdate(prevProps: Props, prevState: State) {
-        const { feedItems: prevFeedItems } = prevProps;
-        const { feedItems: currFeedItems } = this.props;
+        const { feedItems: prevFeedItems, currentUser: prevCurrentUser } = prevProps;
+        const { feedItems: currFeedItems, currentUser: currCurrentUser } = this.props;
```

Comment:
I think currentUser would suffice here. you're differentiating the old one with `prev`